environs/config/config.go

```diff
@@ -177,10 +178,16 @@ const (
 	// FanConfig defines the configuration for FAN network running in the model.
 	FanConfig = "fan-config"
 
-	// CloudInitUserDataKey is the key to specify cloud-init yaml the user wants to add
-	// into the cloud-config data produced by Juju when provisioning machines.
+	// CloudInitUserDataKey is the key to specify cloud-init yaml the user
+	// wants to add into the cloud-config data produced by Juju when
+	// provisioning machines.
 	CloudInitUserDataKey = "cloudinit-userdata"
 
+	// ContainerInheritProperiesKey is the key to specify a list of keys to
```

Comment:
As discussed, let's reword this to remove the cloud init specific implementation aspects

Reply:
updated